resources/assets/js/projects/components/labelTreeListItem.vue

```diff
@@ -8,6 +8,12 @@
     </li>
 </template>
 
+<style scoped>
+.limit-text {
+    word-wrap: break-word;
+}
+</style>
+
```

Comment:
Since you now have a global `limit-text` class (again), this is no longer necessary.

Reply:
Just out of curiosity, do you always know which scss class corresponds to which vue or blade.php file? Or do you use a specific tool to help you identify this? Or are all scss classes accessible in every vue or blade.php file?

Reply:
I don't use any tool. I could just see it from the diff of this PR. The SASS files are globally valid, so if you specify `.limit-text` there without additional constraint, it's valid everywhere. In a Vue single-file component, you should usually use `<style scoped>` for exactly the reason that you do not know where a particular class comes from otherwise. If all styles are scoped (except some global ones), you can always look for the class in the same file.